src/utils/form/validator.ts

```diff
@@ -1,5 +1,6 @@
 const PATTERNS = {
   URL: /^(?:http(s)?:\/\/)?[\w.-]+(?:\.[\w\.-]+)+[\w\-\._~:/?#[\]@!\$&'\(\)\*\+,;=.]+$/,
+  GITHUB_URL: require('is-github-url'),
```

Comment:
Imports like that using old Common JS structure which makes webpack job harder. Use normal import at the top and add type definition i declarations.d.ts file.